package.json

```diff
@@ -95,12 +95,12 @@
     "sass-loader": "^8.0.0",
     "stylelint": "^11.1.1",
     "to-title-case": "^1.0.0",
-    "vue": "^2.5.0",
-    "vue-template-compiler": "2.6.10"
+    "vue": "^2.6.10",
+    "vue-template-compiler": "^2.6.10"
   },
   "peerDependencies": {
     "@mdi/font": "^3.0.39",
-    "vue": "^2.5.x"
+    "vue": "^2.6.10"
```

Comment:
We had an impromptu meeting earlier today because of the versions not being aligned and the build process. We agreed to bump it because the template compiler was fixed to `2.6.10`. It was fixed because of issues with `bili`. So we just aligned it with that version. @kronicker do you have anything else to add or correct?

Reply:
Now I'm confused. You just aligned `vue` and `vue-template-compiler` versions inside dev deps which is fine but didn't you just said latter has/had to be fixed to `2.6.10` due to issue with `bili`?
Also aligning those and fixing them shouldn't have impact on peer deps?

Reply:
/ping @kronicker